docker/docker-windows.go

```diff
@@ -0,0 +1,64 @@
+// +build windows
+
+package docker
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+)
+
+var (
+	errDockerMachineUnknown = fmt.Errorf("unknown DOCKER_MACHINE_NAME")
+)
+
+func (r *Runner) ProjectDir() string{
+	return "/c/Users/dev/GoPath/src/github.com/evo-cloud/hmake"
+}
+
+func (r *Runner) exposeDocker() {
+	r.exposeDockerEnv()
+}
+
+func inspectIds(opt string) (ids []int, err error) {
+	machine := os.Getenv("DOCKER_MACHINE_NAME")
+	if machine == "" {
+		return ids, errDockerMachineUnknown
+	}
+
+	args := []string{"docker-machine", "ssh", machine, "id", opt}
+	cmd := exec.Command(args[0], args[1:]...)
+	cmd.Env = os.Environ()
+	out, err := cmd.Output()
+	if err != nil {
+		return ids, fmt.Errorf("error: [%s]: %v", strings.Join(args, " "), err)
+	}
+	tokens := strings.Split(string(out), " ")
+	for _, token := range tokens {
+		if i, err := strconv.Atoi(strings.TrimSpace(token)); err == nil {
+			ids = append(ids, i)
+		}
+	}
+	if len(ids) == 0 {
+		return ids, fmt.Errorf("no id found: [%s]", strings.Join(args, " "))
+	}
+	return
+}
+
+func currentUserIds() (uid, gid int, grps []int, err error) {
+	var uids, gids []int
+	if uids, err = inspectIds("-u"); err != nil {
+		return
+	}
+	uid = uids[0]
+	if gids, err = inspectIds("-g"); err != nil {
+		return
+	}
+	gid = gids[0]
+	if grps, err = inspectIds("-G"); err != nil {
+		return
+	}
+	return
+}
```

Comment:
Yes, but we are not using docker-machine for Linux. It would be great we can use some `include` mixin feature in Ruby to avoid duplicated codes.